src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -172,6 +179,166 @@ export class BeneficiariesService {
 		}
 	}
 
+	async exportSubjectCsv(req: any, body: any, resp: any) {
+		try {
+			const user = await this.userService.ipUserInfo(req);
+			if (!user?.data?.program_users?.[0]?.organisation_id) {
```

Comment:
add blank lines before and after each `if`

Reply:
added space